libcalico-go/lib/backend/k8s/conversion/workload_endpoint_default.go

```diff
@@ -354,20 +362,52 @@ func handleQoSControlsAnnotations(annotations map[string]string) (*libapiv3.QoSC
 
 	// calico burst annotations
 	if str, found := annotations[AnnotationQoSIngressBurst]; found {
-		ingressBurst, err := parseAndValidateQty(str, *resource.NewQuantity(qosControls.IngressBandwidth, resource.DecimalSI), maxBurst)
+		ingressBurst, err := parseAndValidateQty(str, minBurst, maxBurst)
```

Comment:
So we used to say that burst must be >= bandwidth, but now we don't.  It sounds fine, but are there consequences from changing that?  Maybe a release note?

Reply:
Yes, I believe I had misunderstood things and this restriction is not necessary. No other consequences that I can think of, we should just have a release note and update the docs.